lib/rubycritic/commands/base.rb

```diff
@@ -9,7 +9,7 @@ def initialize(options)
       end
 
       def execute
-        fail NotImplementedError
+        raise NotImplementedError
```

Comment:
After reading through that last link, it turns out the tides are shifting!
These are quite recent changes but it suggests to move away from fail - Rubocop 0.37.0 does just that.

So I changed my mind here, we should stick with `raise`.

Personal preferences are over-rated :P